src/PDOutline.jl

```diff
@@ -0,0 +1,407 @@
+export PDOutline,
+    PDOutlineItem,
+    PDOutlineIter,
+    item_level,
+    items_count,
+    items
+
+using LabelNumerals
+using RomanNumerals
+using ..Cos
+
+####################################################################################
+## Data Structures
+"""
+```
+    PDOutlineItem
+```
+Representation of PDF document Outline item.
+
+It is currently defined as `Dict{Symbol, Any}`.
+Check description of function `pdDocGetOutline` for more information.
+
+Methods which operate on this structure:
+- `item_level(item::PDOutlineItem)` - return nesting level of item inside whole Outline.
+"""
+PDOutlineItem = Dict{Symbol, Any}
```

Comment:
Table 153 PDF Spec pretty much outlines the interface for the outline items. This structure does not quite look representative of the same.

Reply:
May You please little more elaborate.
I think I choose stricture that do not loose any important information yet be easy to use.